src/main/java/com/gengo/client/GengoConstants.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * 
+ */
+package com.gengo.client;
+
+/**
+ * @author hiroyu-n@inter-plug.co.jp
+ *
+ */
+public final class GengoConstants {
+	protected static String BASE_URL_STANDARD = "https://api.gengo.com/v2/";
```

Comment:
I don't think we want to derive something from this class.
So I guess package private is better than `protected`.